### ISSUE TITLE
some files just hang Jellyfin no playback or error

### ISSUE THREAD

**Issue by Stooovie** [jellyfin/jellyfin-tizen#206]
Here's the log: 

https://transfer.sh/VqzkmJknuo/Log.txt.zip

It's the "It's Always Sunny in Philadephia" file. UI just shows the poster in the BG, no UI, no spinner, no error code. The only thing I can do is exit the JF app with my remote. Most other files play. Setting transcoding on/off does not fix the issue. 

File plays just fine in Jellyfin apps and webapps on other devices.

Jellyfin for Tizen 0.1.0 built from source a week ago (june 2023).

**Comment by dmitrylyzo** [comment]
According to the log, there was an attempt to play the file (MKV) directly.

Please provide your TV model and media info.

**Comment by dmitrylyzo** [comment]
The problem is `H264 10bit` - this combo isn't well supported in general.

I have the same issue on NU7400, iirc. Need to recheck.

**Comment by Stooovie** [comment]
I understand, does JF for Tizen have some sort of look-up table with TV model capabilities? It should be definitely handled better - now it just hangs on the spinner indefinitely without as much as an error message.

**Comment by dmitrylyzo** [comment]
[Here](https://github.com/jellyfin/jellyfin-web/blob/6304e27940e59d41ba9de41f9f0d5da7392cffbb/src/scripts/browserDeviceProfile.js#L785) is just a naive check.

If there is no Tizen TV that supports H264 10bit, we just need to remove `browser.tizen`. But `canPlayType` may still give a false positive result.

**Comment by Mauroq** [comment]
I have tested that this fixes the issue on my q95. I couldn't find any Samsung tizen TVs that specifically mention support for h264 10bit.

**Comment by JClarke90** [comment]
I've been having this issue too on NU7100

I rebuilt jellyfin with the latest code today but still getting issues when playing h264 High 10.

Looks like its trying to play direct according to jellyfin dashboard but the TV doesn't support it.

![image](https://github.com/jellyfin/jellyfin-tizen/assets/12259768/04721550-a233-4e4b-8bd0-a9b16ec3feb2)

**Comment by dmitrylyzo** [comment]
> I rebuilt jellyfin with the latest code today

That PR was not backported to `master` (_Note the `stable-backport` label on it_).

**Comment by JClarke90** [comment]
Yep, I realised 2 mins after posting, just rebuilding with that build now. Thank you.

**Comment by JClarke90** [comment]
@dmitrylyzo 
So I've tried to build from branch release-10.8.z  but keep running into an issue, wondered if you can help? 

 I'm fairly new to this but it looks like it wants a newer version of node, is that something that needs up dating in the code as I don't even have node installed on my box.

![image](https://github.com/jellyfin/jellyfin-tizen/assets/12259768/69d65217-66b2-4853-8ff0-dbd65489f4b2)


Otherwise I can post a new issue in the main jellyfin-web

**Comment by dmitrylyzo** [comment]
@JClarke90 it looks like you are still building `10.9.0` (`master`), since `JELLYFIN_WEB_VERSION=master`. `master` now requires modern NodeJS (_But not really - you can downgrade it for now_). This should be fixed in the repo from which the Dockerfile is (_This is not our repo_).
`10.8.z` has no such requirement.

It is better to ask questions about the build process in chat or on the forum (https://jellyfin.org/contact), as they are individual (missing step, missing package, wrong command, etc.).

**Comment by JClarke90** [comment]
Just want to confirm this also fixed the issue on my NU7100, once I managed to build the correct version.

**Comment by Stooovie** [comment]
> Just want to confirm this also fixed the issue on my NU7100, once I managed to build the correct version.

Which is?

**Comment by JClarke90** [comment]
> > Just want to confirm this also fixed the issue on my NU7100, once I managed to build the correct version.
> 
> Which is?

Release-10.8.z

**Comment by AwkwardGecko** [comment]
I had a similar issue, so I thought I'd add this here for convenience: 

It may be an issue with Jellyfin sharing a cache with the default Samsung web browser. I managed to fix it by opening the web browser > privacy and security > delete browsing data.

Hope this helps someone else :)
